style-guide/about-style-guide.md

```diff
@@ -0,0 +1,168 @@
+# The style guide
+
+The style guide template itself occasionally includes placeholder text that
+is intended for you to replace with your own text. This text is indicated in
+{curly brackets}.
+
+## When do I need a style guide?
+
+A style guide provides project contributors with general guidelines for writing
+project documentation. The overall goal of a style guide is to ensure quality
+and consistency throughout the project’s documentation, which is especially
+important if different authors are contributing to the documentation over time.
+
+A style guide can provide early guidance to new contributors as they make
+decisions related to your project’s documentation. Contributors often have to
+make subjective judgment calls about the documentation, such as:
```

Comment:
The phrase "judgement calls" might be slang. I suggest "decisions" instead.

Reply:
I like that suggestion! I've adopted it.